frontend/fingertips-frontend/components/pages/chart/index.tsx

```diff
@@ -0,0 +1,29 @@
+'use client';
+
+import { LineChart } from '@/components/linechartsOptions';
+import { WeatherForecast } from '@/generated-sources/api-client';
+import { H1 } from 'govuk-react';
+import { PlainTable } from '@/components/table';
+
+type ChartProps = {
+  data: WeatherForecast[];
+};
+
+export function Chart({ data }: Readonly<ChartProps>) {
+  return (
+    <>
+      <H1>Line Chart</H1>
+      <div>
```

Comment:
Given the `LineChart` and `PlainTable` components already wrap a `<div>` around everything within them, are the `div`s wrapped around the components at this level redundant?